test/e2e/extended_suite_test.go

```diff
@@ -18,6 +19,14 @@ var (
 
 var _ = BeforeSuite(func() {
 	c = newTestClient(*kubeconfig)
+
+	if _, err := os.Stat("../../_data/_out/dedicated-cluster-admin.kubeconfig"); !os.IsNotExist(err) {
```

Comment:
Read config.GinkgoConfig.FocusString from "github.com/onsi/ginkgo/config" and fail if one of these kubeconfigs does not exist and the focus string matches DedicatedAdmin (example [here](https://github.com/openshift/openshift-azure/blob/58f4b1e8efc8cb9b9e582039c1f6009eb89d7d25/test/e2erp/client_test.go#L45-L55)).

Reply:
Fixed.